examples/serial/board/imx8mm_evk/serial.system

```diff
@@ -2,12 +2,12 @@
 <system>
     <memory_region name="uart" size="0x1_000" phys_addr="0x30890000" />
 
-    <memory_region name="rx_data_driver" size="0x200_000" page_size="0x200_000" />
-    <memory_region name="tx_data_driver" size="0x200_000" page_size="0x200_000" />
-    <memory_region name="rx_data_client0" size="0x200_000" page_size="0x200_000" />
-    <memory_region name="tx_data_client0" size="0x200_000" page_size="0x200_000" />
-    <memory_region name="rx_data_client1" size="0x200_000" page_size="0x200_000" />
-    <memory_region name="tx_data_client1" size="0x200_000" page_size="0x200_000" />
+    <memory_region name="rx_data_driver" size="0x2_000" page_size="0x1_000" />
+    <memory_region name="tx_data_driver" size="0x2_000" page_size="0x1_000" />
+    <memory_region name="rx_data_client0" size="0x2_000" page_size="0x1_000" />
+    <memory_region name="tx_data_client0" size="0x2_000" page_size="0x1_000" />
+    <memory_region name="rx_data_client1" size="0x2_000" page_size="0x1_000" />
+    <memory_region name="tx_data_client1" size="0x2_000" page_size="0x1_000" />
```

Comment:
You can remove the `page_size` from all of these since the default is 0x1000

Reply:
Done